lms/data_tasks/report/create_from_scratch/05_entities_decorated/01_users/01_create_view.jinja2.sql

```diff
@@ -44,15 +83,32 @@ CREATE MATERIALIZED VIEW report.users AS (
         user_details.display_name,
         user_details.email,
         raw_users.username,
-        CASE
-            WHEN user_details.is_teacher IS true THEN true ELSE false
-        END AS is_teacher,
         -- Add a column which indicates that a user is a teacher in at least
         -- one context. This helps us notice / demonstrate we are only keeping
         -- teacher contact info. Keeping student contact info out of region is
         -- not permitted.
-        raw_users.registered_date
+        CASE
+            WHEN user_details.is_teacher IS true THEN true ELSE false
+        END AS is_teacher,
+        -- Daily resolution is fine. Some of these are only accurate to the week
+        LEAST(
+            raw_users.registered_date,
+            user_modify_dates.created_date
+        )::DATE AS created_date,
+        user_modify_dates.updated_date::DATE,
```

Comment:
Should we even have this? I'm not sure what we'd use it for. It seems weird not to have an updated date for a created date.

It does seem superceeded by the `last_activity_date`.

Reply:
If I understood this `updated_date` comes from lms.user.updated which I reckon only changes when the roles are updated there and we know that the roles in that table are useless so it might become immutable if we fix that.